backend/tests/Tests/Cron.Tests.fs

```diff
@@ -48,9 +48,8 @@ let testCronSanity =
   testTask "cron sanity" {
     let! canvasID = initializeTestCanvas "cron-sanity"
 
-    let! e = p "5 + 3"
-    let resultExpr =
-      C2DT.Result.fromDT PT2DT.Expr.fromDT e PT2DT.TypeReference.fromDT
+    let! e = p "5 ++ 3"
+    let resultExpr = C2DT.Result.fromDT PT2DT.Expr.fromDT e RT.RuntimeError.fromDT
```

Comment:
could you please check this?
I am not sure if `RT.RuntimeError.fromDT` is the right function to send here